Mathlib/Combinatorics/SimpleGraph/DegreeSum.lean

```diff
@@ -113,6 +108,21 @@ theorem sum_degrees_eq_twice_card_edges : ∑ v, G.degree v = 2 * G.edgeFinset.c
   G.dart_card_eq_sum_degrees.symm.trans G.dart_card_eq_twice_card_edges
 #align simple_graph.sum_degrees_eq_twice_card_edges SimpleGraph.sum_degrees_eq_twice_card_edges
 
+/-- The complete graph on `n` vertices has `n.choose 2` edges. -/
+theorem top_card_edges_eq_card_choose_two :
```

Comment:
```suggestion
theorem card_edgeFinset_top_eq_card_choose_two :
```

Reply:
Though maybe the naming is inconsistent in this area of mathlib, and @YaelDillies or @b-mehta knows better

Reply:
I agree with your name, Eric.